healthScore/healthScore/views.py

```diff
@@ -56,3 +58,28 @@ def add_mock_data(request):
         return HttpResponse("Data Added to the database")
     else:
         return HttpResponse("Please change the request method to POST")
+
+@csrf_exempt
+def register(request):
+    if request.method == 'POST': # when the form is submitted
+        email = request.POST.get('email')
+        username = request.POST.get('username')
+        password = request.POST.get('password')
+        fullname = request.POST.get('fullname')
+        dob = request.POST.get('dob')
+        gender = request.POST.get('gender')
+        street_address = request.POST.get('street_address')
+        city = request.POST.get('city')
+        state = request.POST.get('state')
+        phone_number = request.POST.get('phone_number')
+        identity_proof = request.POST.get('identity_proof')
+
+        if not user.objects.filter(email=email).exists():
+            user.objects.create(email=email, userName=username, password=password, name=fullname, 
+                                dob=dob, gender=gender, address=street_address, contactInfo=phone_number)
```

Comment:
Here, we are storing the `address` as `street_address`. Let's concatenate the `city` and `state` fields as well, or take only one input from the user for address?

Reply:
It is a problem. It was a workaround to run the app. Since the fields on the registration page don't exactly match the database, I was thinking we could modify both parts to make them consistent in the next sprint.

Reply:
Ohh right. Is something like this possible?
`address = street_address + ', ' + city + ', ' + state`